addons/binding/org.openhab.binding.nikohomecontrol/OSGI-INF/NikoHomeControlHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
I started development in 2016. I manually changed dates in all of them as mvn license:format does not seem to work in my environment.